server/services/participants.js

```diff
@@ -177,6 +181,7 @@ const updateParticipant = async (participantInfo) => {
       );
       // ensure that a participant has a cohort before adding post hire status
       if (cohort && cohort.length > 0 && graduationStatuses.length === 0) {
+        await updatePostHireStatus({ participantId: participantInfo.id });
```

Comment:
Do we need to update? check, in line 183
if (cohort && cohort.length > 0 && `graduationStatuses.length === 0)`
there is no post-hire graduation statuses for participant